frontend/language/src/nb.json

```
@@ -868,6 +868,7 @@
   "ux_editor.component_radio_button": "Radioknapp",
   "ux_editor.component_text_area": "Langt svar",
   "ux_editor.component_unknown": "Ukjent komponent",
+  "ux_editor.component_confirm_delete": "Er du sikkert på at du vil slette komponenten?",
```

I'm wondering now if we should not rather say `denne komponenten` as we did for `dette språket` in `language_display_confirm_delete`:
```suggestion
  "ux_editor.component_confirm_delete": "Er du sikkert på at du vil slette denne komponenten?",
```

---

done